### PR TITLE
Tweak Default DI Config

### PR DESCRIPTION
**Pull request by gsteel**
Alias `Zend\HttpHandlerRunner\RequestHandlerRunner` to `SwooleRequestHandlerRunner` so that you can target a Swoole environment specifically with a delegator factory.

I don't know if this affects DIC's other than Zend Service Manager?

### CODE REVIEW

File: src/ConfigProvider.php [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 use Psr\Http\Message\ServerRequestInterface;
 use Zend\HttpHandlerRunner\RequestHandlerRunner;
+use function extension_loaded;
```
Comment (snapshotpl) [inline src/ConfigProvider.php]: ?

Comment (gsteel) [reply src/ConfigProvider.php]: Why not?

Comment (webimpress) [reply src/ConfigProvider.php]: @snapshotpl check the whole file, the function is used in line 20.

**Comment by weierophinney** [comment]
> I don't know if this affects DIC's other than Zend Service Manager?

Alias support is a requirement for containers used with Expressive ([see the alias documentation](https://docs.zendframework.com/zend-expressive/v3/features/container/config/#aliases)), so this change makes sense to me.

**Comment by weierophinney** [comment]
Thanks, @gsteel! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**